root/edit/details/AddReleaseGroup.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+import React from 'react';
+
+import DescriptiveLink from
+  '../../static/scripts/common/components/DescriptiveLink';
+import ExpandedArtistCredit from
+  '../../static/scripts/common/components/ExpandedArtistCredit';
+
+type AddReleaseGroupProp = {
+  ...EditT,
+  +display_data: {
+      +artist_credit: ArtistCreditT,
+      +comment?: string,
+      +name: string,
+      +release_group: ReleaseGroupT,
+      +secondary_types?: string,
+      +type?: ReleaseGroupTypeT,
+  },
+};
+
+const AddReleaseGroup = ({edit}: {edit: AddReleaseGroupProp}) => {
+  const display = edit.display_data;
+  const type = display.type;
+  return (
+    <>
+      <table className="details">
+        <tbody>
+          <tr>
+            <th>{l('Release Group:')}</th>
+            <td>
+              <DescriptiveLink entity={display.release_group} />
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <table className="details add-release-group">
+        <tbody>
+          <tr>
+            <th>{l('Name:')}</th>
+            <td>{display.name}</td>
+          </tr>
+
+          <tr>
+            <th>{l('Artist:')}</th>
+            <td>
+              <ExpandedArtistCredit
+                artistCredit={display.artist_credit}
+              />
+            </td>
+          </tr>
+
+          {display.comment ? (
+            <tr>
+              <th>{addColon(l('Disambiguation'))}</th>
+              <td>{display.comment}</td>
+            </tr>
+          ) : null}
+
+          {type ? (
+            <tr>
+              <th>{l('Primary Type:')}</th>
+              <td>{type.name}</td>
+            </tr>
+          ) : null}
+
+          {display.secondary_types ? (
+            <tr>
+              <th>{l('Secondary Types:')}</th>
+              <td>{display.secondary_types}</td>
```

Comment:
These should also be translated right now, but that's a bit trickier, since display_data only gets a string joining the names. We should probably build the string here (using lp_attributes to translate each type name), and change Edit::ReleaseGroup::Create to pass an array of types or an array of type names instead. @mwiencek: what do you think?

Reply:
Hmm, I'd probably just change the `->name` to `->l_name` in display data for now. I get that we want a consistent place to do the translations, but we don't really have any plans to remove the Perl i18n later any time soon either, so it's just avoiding unnecessary work for now to get the core React conversion merged.